test_common/page_objects/et1/group_claims_upload_page.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './base_page'
+module EtFullSystem
+  module Test
+    module Et1
+      class GroupClaimsUploadPage < BasePage
+        section :main_content, '#content .main-section .main-content' do
+          section :group_claims, :xpath, (XPath.generate { |x| x.descendant(:fieldset)[x.descendant(:legend)[x.string.n.is("People making a claim with you")]] }) do
+            section :file_upload, :field, 'Upload spreadsheet (optional)' do
+              def set(value)
+                browser = page.driver.browser
```

Comment:
This probably needs some explaining.
When using a remote driver for selenium, it needs a 'file detector'.  The file detector is responsible for detecting which files are local and which are remote.  Here, we are saying they are all remote so they all get uploaded.  But, because we are changing global state in the browser - then we put it back afterwards.

Note that reading this value is not part of a public API - so am having to use the send method - not very nice - but I wasn't happy with changing it without putting it back.